src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1952,63 +1952,44 @@ because xbuild doesn't support framework reference assemblies.
   <!-- Create our intermediate directory for assemblies -->
   <MakeDir Directories="$(MonoAndroidIntermediateAssemblyDir)" Condition=" !Exists ('$(MonoAndroidIntermediateAssemblyDir)') " />
 </Target>
+
+<Target Name="_CollectPlatformJarFilenames">
+	<ItemGroup>
+		<_PlatformJars Include="mono.android.jar" />
+		<_PlatformJars Include="runtime.jar" />
+	</ItemGroup>
+</Target>
   
-<Target Name="_GetMonoPlatformJarPath">
-  <GetMonoPlatformJar TargetFrameworkDirectory="$(TargetFrameworkDirectory)">
-    <Output TaskParameter="MonoPlatformJarPath" PropertyName="MonoPlatformJarPath" />
-    <Output TaskParameter="MonoPlatformDexPath" PropertyName="MonoPlatformDexPath" />
-  </GetMonoPlatformJar>
+<Target Name="_GetMonoPlatformJarPath" DependsOnTargets="_CollectPlatformJarFilenames">
+  <GetMonoPlatformJars TargetFrameworkDirectory="$(TargetFrameworkDirectory)" JarFiles="@(_PlatformJars)">
+    <Output TaskParameter="MonoPlatformJarPaths" ItemName="MonoPlatformJarPaths" />
```

Comment:
Should these outputs be `_`-prefixed?

Reply:
It should. That does bring up a question though.... `MonoPlatformJarPath` was a public property (even though it should not have been I guess? ) can we remove it?